### PR TITLE
build: prepare v0.18.0.0

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,8 +136,8 @@ Dates are provided in the format YYYY-MM-DD.
 | 1788000                        | 2019-03-09 | v10               | v0.14.0.0              | v0.14.1.2                  | New PoW based on Cryptonight-R, new block weight algorithm, slightly more efficient RingCT format
 | 1788720                        | 2019-03-10 | v11               | v0.14.0.0              | v0.14.1.2                  | forbid old RingCT transaction format
 | 1978433                        | 2019-11-30 | v12               | v0.15.0.0              | v0.16.0.0                  | New PoW based on RandomX, only allow >= 2 outputs, change to the block median used to calculate penalty, v1 coinbases are forbidden, rct sigs in coinbase forbidden, 10 block lock time for incoming outputs
-| 2210000                        | 2020-10-17 | v13               | v0.17.0.0              | v0.17.3.2                  | New CLSAG transaction format
-| 2210720                        | 2020-10-18 | v14               | v0.17.1.1              | v0.17.3.2                  | forbid old MLSAG transaction format
+| 2210000                        | 2020-10-17 | v13               | v0.17.0.0              | v0.18.0.0                  | New CLSAG transaction format
```
Comment (hyc) [inline README.md:139]: Why did these two lines change?

**Comment by plowsof** [comment]
i ran this command on my own blockchain lmdb file (providing the correct `--data-dir` if needed):
```
./monero-blockchain-export --output-file checkpoints.dat --block-stop 2661600 --blocksdat
```
confirmed the checksum of the checkpoints file matches that of this PR:
```
$ shasum -a256 checkpoints.dat
e9371004b9f6be59921b27bc81e28b4715845ade1c6d16891d5c455f72e21365  checkpoints.dat
```
also confirmed the hash of blockheight 2661600 has the correct hash listed [here](https://xmrchain.net/search?value=41c9060e8426012238e8a26da26fcb90797436896cc70886a894c2c560bcccf2)

LGTM

**Review by plowsof — APPROVED** [review]

**Review by jeffro256 — APPROVED** [review]
Checkpoints.dat checks out, as well as the various misc changes

**Review by hundehausen — COMMENTED** [review]
Lines 141 + 142 of README.md must be changed to the new dates

**Review by nahuhh — CHANGES_REQUESTED** [review]
README.md line 141/142 have incorrect heights and dates
Example of correction: https://github.com/monero-project/monero/pull/8431

**Comment by hundehausen** [comment]
Here is a new [PR](https://github.com/selsta/monero/pull/1)

**Review by nahuhh — APPROVED** [review]
Looks good

**Pull request merged**